packages/FE/src/pages/quiz-create/index.tsx

```diff
@@ -1,10 +1,115 @@
+import { useState } from 'react';
+
+import { CustomButton } from '@/shared/ui/buttons';
+import PlusIcon from '@/shared/assets/icons/plus.svg?react';
 import QuizCreateSection from './ui/QuizCreateSection';
 
+interface Choice {
+  content: string;
+  isCorrect: boolean;
+  position: number;
+}
+
+export interface QuizData {
+  content: string;
+  quizType: 'MC' | 'TF';
+  timeLimit: number;
+  point: number;
+  position: number;
+  choices: Choice[];
+}
+
 export default function QuizCreatePage() {
-  //TODO: 문제 추가하기 상태 관리
+  const [currentQuizIndex, setCurrentQuizIndex] = useState(0);
+  const [quizzes, setQuizzes] = useState<QuizData[]>([
+    {
+      content: '',
+      quizType: 'MC',
+      timeLimit: 30,
+      point: 1000,
+      position: 0,
+      choices: [
+        { content: '', isCorrect: false, position: 0 },
+        { content: '', isCorrect: false, position: 1 },
+      ],
+    },
+  ]);
+
+  const addNewQuiz = () => {
+    setQuizzes((prev) => [
+      ...prev,
+      {
+        content: '',
+        quizType: 'MC',
+        timeLimit: 30,
+        point: 1000,
+        position: prev.length,
+        choices: [
+          { content: '', isCorrect: false, position: 0 },
+          { content: '', isCorrect: false, position: 1 },
+        ],
+      },
+    ]);
+    setCurrentQuizIndex((prev) => prev + 1);
+  };
+
+  const handlePreQuiz = () => {
+    if (currentQuizIndex === 0) return;
+    setCurrentQuizIndex((prev) => prev - 1);
+  };
+
+  const handleNextQuiz = () => {
+    if (currentQuizIndex === quizzes.length - 1) return;
+    setCurrentQuizIndex((prev) => prev + 1);
+  };
+
   return (
-    <div className="flex w-full">
-      <QuizCreateSection />
+    <div className="flex flex-col w-full mt-6 mr-6">
+      <div className=" flex gap-4 bg-white rounded-base p-4 mb-4">
+        <button className="text-weak-md" onClick={handlePreQuiz}>
+          이전 문제
+        </button>
+        <button className="text-weak-md" onClick={handleNextQuiz}>
+          다음 문제
+        </button>
+        <div className="flex-1 flex justify-end text-weak-md">문제 유형</div>
+      </div>
+
+      <QuizCreateSection
+        key={currentQuizIndex}
+        currentQuizIndex={currentQuizIndex}
+        quizData={quizzes[currentQuizIndex]}
+        onQuizUpdate={(updatedData) => {
+          setQuizzes((prev) => {
+            const newQuizzes = [...prev];
+            newQuizzes[currentQuizIndex] = updatedData as QuizData;
+            return newQuizzes;
+          });
```

Comment:
타입 단언하는 것보다 `updatedData`의 타입을 지정해주는 게 훨씬 좋을 거 같아요!
```suggestion
        onQuizUpdate={(updatedData: QuizData) => {
          setQuizzes((prev) => {
            const newQuizzes = [...prev];
            newQuizzes[currentQuizIndex] = updatedData;
            return newQuizzes;
          });
```